examples/trivial-digital-ocean.nix

```diff
@@ -0,0 +1,13 @@
+{
+  resources.sshKeyPairs.ssh-key = {};
+
+  machine = { config, pkgs, ... }: {
+    services.nginx.enable = true;
+    networking.firewall.allowPing = true;
```

Comment:
Is this absolutely necessary to deploy DO? iirc `allowPing` is true by default since 16.03